templates/dynflow_worker.yml.erb

```diff
@@ -0,0 +1,5 @@
+:concurrency: <%= scope['concurrency'] %>
+:queues:
```

Comment:
@ezr-ondrej @adamruzicka I don't know if you have considered or if it would be possible, but a future looking idea that could be nice is making the queues a setting that's changeable without restart. This would simplify the installer support, make the workers less reliant on a config file and restarts and potentially give the ability to dynamically scale and assign out queues based on workload from the application itself via user config or "smarts".

Reply:
It would be nice, but I'm not sure if sidekiq supports changing queue configuration without restarting the worker. Still, might be worth looking into it